index.html

```diff
@@ -3141,27 +3151,67 @@ <h3 id="applying">
       </h2>
       <p>
         Each <dfn>application object</dfn> represents an application related to
-        the web application. An application object has three properties: a
-        <code>platform</code> which represents the platform it is associated
-        to, a <code>url</code> which represents the URL where the application
-        can be found and an <code>id</code> which can be used as an information
-        additional to the URL or instead of the URL, depending on the platform.
+        the web application. An application object has five properties:
```

Comment:
Can we change this to "has the following properties" so we don't have to remember to keep this number up to date as we add more?